conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+# Research Software Engineering Conference 2019
+
+* Conference website: https://rse.ac.uk/conf2019/
+* Application form: https://docs.google.com/forms/d/e/1FAIpQLScVml19n44QfuOfjuaRlJUtXNMih-X4xUrNjlr8XJVHeyTyQg/viewform
+* Location: University of Birmingham
+* Date: 17th - 19th September 2019
+
+### Workshop Length
+
+180 mins (to allow time for Kubernetes cluster deployment and an introduction to reproducible research and BinderHub).
+
+### Title
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
```

Comment:
Do you need a helper for this @sgibson91? @pherterich, are you able to attend to help out? Or @martintoreilly?

Reply:
@annakrystalli Did you mention that you would be going?

Reply:
I'll be there!

Reply:
Would you mind being a mentor/helper for my session (if it's accepted 🤞) ?

Reply:
Sure thing! (Although I feel it's probably going to be you mentoring me to be able to be a decent  helper on this! 😜). Also just a heads up that I'm putting in a proposal for a short walk-through too so if that gets accepted, we should let the organisers know so they don't schedule a clash.

Reply:
You're a star! 🌟I honestly think just having another pair of boots on the ground even to attract my attention if I get distracted presenting would be helpful 😂

Reply:
I might be volunteering again and would make a case to be the volunteer in that session then. No idea if that's helpful except for having familiar faces in the room :smile:

Reply:
I will always appreciate a friendly face to cheerlead 😊

Reply:
Please do add Patricia and Anna - that's a great team and we can always find additional helpers if they're not able to join when the schedule is released 👾